app/views/api-keys/create.njk

```diff
@@ -48,7 +48,7 @@ Create an API key - {{currentService.name}} {{currentGatewayAccount.full_type}}
     </p>
   {% else %}
     <h1 class="govuk-heading-l page-title">New API key</h1>
-    <p class="govuk-body">Copy your key to somewhere safe. You won’t be able to see it again once you leave this page.</p>
+    <p class="govuk-body">Copy your key to somewhere safe. You won’t be able to see it again once you leave this page. You must copy the whole key, including the api_test_ or api_live_ prefix.</p>
```

Comment:
Do we want to use a `<code></code>` tag around the `api_test_` and `api_live_` prefix parts to differentiate them from the rest of the paragraph? (as in this comment!)